### DISCUSSION TITLE
Adding NE555 to Qucs-s?

### DISCUSSION THREAD

**Discussion by 5iemen5** [ra3xdh/qucs_s#202] [Q&A] [answered]
I'm new to SPICE engine, and have no idea how to install, for example, a component model of NE555 timer. LTSpice shares the same engine, I believe. But it has NE555 model as a native component, and Qucs-s situates to not have one. What should one do in this case?

**Comment by ra3xdh** [comment]
Designing of the device model libraries is not a complex task but requires some time. I am planning to add 555 timer library in the next release. You may also look at #120 and find models that works with Qucs-S+Ngspice.

**Comment by ra3xdh** [comment]
Also look at this manual how to attach an existing SPICE model to Qucs-S: https://qucs-s-help.readthedocs.io/en/latest/SubLib.html#using-manufacturers-component-data-libraries LTSpice models have some incompatible features and may require manual editing. See #141

**Comment by ra3xdh** [comment, marked as answer]
The library containing 555 timer model has been added since v1.0.1. See #120
